travis/install-wdl.sh

```diff
@@ -17,9 +17,9 @@ cached_fetch_jar_from_github () {
 	ln -s $CACHE_DIR/$_jar_fname $_tool_name.jar
 }
 
-cached_fetch_jar_from_github broadinstitute cromwell womtool 30.2
-cached_fetch_jar_from_github broadinstitute cromwell cromwell 30.2
-cached_fetch_jar_from_github dnanexus dxWDL dxWDL 0.60.2
+cached_fetch_jar_from_github broadinstitute cromwell womtool 32
+cached_fetch_jar_from_github broadinstitute cromwell cromwell 32
+cached_fetch_jar_from_github dnanexus dxWDL dxWDL 0.66.2
```

Comment:
If it doesn't cause much trouble, since you're updating from dxWDL 0.60.2 -> 0.66.2 anyway, want to just try going all the way to 0.68.1 (since some time has elapsed in the course of the development of this branch)?

Reply:
Let's see what happens. I'll try updating Cromwell too.